shared_model/backend/protobuf/commands/proto_add_asset_quantity.hpp

```diff
@@ -20,7 +20,7 @@
 
 #include "interfaces/commands/add_asset_quantity.hpp"
 
-#include "backend/protobuf/common_objects/amount.hpp"
+#include "backend/protobuf/common_objects/proto_amount.hpp"
```

Comment:
Why?

Reply:
We have no convension about naming. It causes such situations. We probably need to decide on it asap

Reply:
Yeh, we are. But still there's should be some meaningful reason for previous name change. And personally I'd like to this fix being reverted

Reply:
Probably in separate PR we should fix it

Reply:
It was just unification. All files in protobuf directory start their names with proto and this one was only an exception.

Reply:
It's common_objects, they definetely has no connection to protobuf

Reply:
It's proto implementation of common objects.